arches/app/templates/registration/password_reset_form.html

```diff
@@ -1,26 +1,31 @@
-
-
 {% extends "registration/base.html" %}
 {% load i18n %}
 {% load template_tags %}
 {% block title %}{% trans "Password recovery" %}{% endblock %}
 {% block content %}
 <h3 class="login-panel-header arches-signin">{% trans "Password Recovery" %}</h3>
-	<form method="post" action="{{ url }}">
-		<div class="form-group">
-
-			{% csrf_token %}
-			<div class="form-group">
-				<div class="input-group">
-					<div class="input-group-addon"><i class="text-lg ion-log-in"></i></div>
-					{{ form.email.errors }}
-					{{ form.email}}
-				</div>
-			</div>
-			<div class="arches-signin-btn">
-				<button class="btn btn-primary btn-lg btn-block" type="submit">{% trans "Recover my password" %}</button>
-			</div>
-		</div>
+    <form method="post" action="{{ url }}">
+        <div class="form-group">
 
-	</form>
+            {% csrf_token %}
```

Comment:
The password reset form has a lot of empty space that pushes it below the bottom of the page. 
<img width="253" alt="image" src="https://github.com/archesproject/arches/assets/1877663/f54fad58-1ae9-42f8-886d-b9ea994c3bba">

<img width="253" alt="image" src="https://github.com/archesproject/arches/assets/1877663/2f59c2aa-9651-449d-ae9a-ff01474590b5">